css-contain-2/Overview.bs

```diff
@@ -1970,9 +1989,13 @@ Restrictions and Clarifications {#cv-notes}
 		and not cause any forced layouts.
 	</div>
 
-4. The initial determination of visibility for ''content-visibility: auto'' must
-    happen in the same frame that determined an existence of a new
-    ''content-visibility: auto'' element.
+4. Elements with ''content-visibility: auto''
+	that have not determined [=proximity to the viewport=]
+	must determine their proximity to the viewport
+	in the next <a href="https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering">Update the Rendering</a> cycle. 
+	The effect of this determination must be reflected in the visual update which results from this Update the Rendering cycle.
```

Comment:
Probably good to link this "Update the Rendering" too.

And maybe it should use a bikeshed link-definition at the top rather than using `<a href=''>...</a>` syntax.

Reply:
Turns out it's exported! I've changed all of these to `[=update the rendering=]`